specification/source/verification.rst

```diff
@@ -383,3 +387,306 @@ Despite these differences, the computed and the simulated control signals
 show good agreement, which is also demonstrated
 by verifying the time series with the
 funnel software, whose output is shown in :numref:`fig_coo_coi_val_fun`.
+
+
+Specification for automating the verification
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The example :numref:`sec_ver_exa` describes a manual process of composing
+the verification model and executing the verification process.
+In this section, we provide specifications for how this process can be automated.
+The automated workflow uses the same modules as in :numref:`sec_ver_exa`,
+except that the unit conversion will need to be done by the tool that reads
+the CSV files and sends data to the Building Automation System, and that reads
+data from the Building Automation System and writes them to the CSV files.
+This design decision has been done because CDL provides all required unit information,
+but this is not the case in general for a building automation system.
+Moreover, in the process described in this section,
+the CSV files will be read directly by the Modelica simulation
+environment rather than using the CSV file reader described in :numref:`sec_ver_csv_reader`.
+
+Use cases
+~~~~~~~~~
+
+We address two use cases. Both uses cases verify conformance of the time series
+generated by a control control sequence specified in CDL against the time series
+of an implementation of a real controller.
+For both use cases, the precondition is that one control sequence,
+or several control sequences, are available in CDL.
+The output will be a report that describes whether the real implementation
+conforms to the CDL implementation within a user-specified error tolerance.
+The difference between the two uses cases is as follows:
+In scenario 1, the CDL model contains the controller and blocks that generate the control input
+time series.
+The time series from this CDL model will be used to test the real controller.
+In scenario 2, data trended from a real controller will be used to
+verify the controller against its CDL specification.
```

Comment:
clarify that those are outputs of both CDL and "real" controller.

Reply:
reformulated.